server/app/services/rpc_server.rb

```diff
@@ -128,15 +190,9 @@ def handling_instance(grid_id, name)
     @handlers[grid_id][name]
   end
 
-  # @param [Faye::Websocket] ws
-  # @param [Object] message
-  def send_message(ws, message)
-    EM.next_tick { # important to push sending back to EM reactor thread
-      ws.send(MessagePack.dump(message.as_json).bytes)
```

Comment:
The `RpcServer` class shouldn't need to know anything about `EventMachine`, the `Faye::Websocket` or `MessagePack`...